test/javascripts/acceptance/chat-quoting-test.js

```diff
@@ -0,0 +1,235 @@
+import { test } from "qunit";
+import { click, currentURL, visit } from "@ember/test-helpers";
+import {
+  acceptance,
+  exists,
+  query,
+  visible,
+} from "discourse/tests/helpers/qunit-helpers";
+import {
+  chatChannels,
+  chatView,
+} from "discourse/plugins/discourse-chat/chat-fixtures";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+let quoteResponse = {
+  bbcode: `[chat quote="martinchat;3875498;2022-02-04T01:12:15Z" channel="The Beam Discussions"]
+  an extremely insightful response :)
+  [/chat]`,
+};
+
+function setupPretenders(server, helper) {
+  server.get("/chat/chat_channels.json", () => helper.response(chatChannels));
+  server.post(`/chat/9/quote.json`, () => helper.response(quoteResponse));
+  server.post(`/chat/4/quote.json`, () => helper.response(quoteResponse));
+  server.post(`/chat/7/quote.json`, () => helper.response(quoteResponse));
+  server.get("/chat/:chatChannelId/messages.json", () =>
```

Comment:
```suggestion
  server.get("/chat/:chat_channel_id/messages.json", () =>
```

Minor but kind of odd since this isn't a JS variable.

Reply:
Just copied it from another test, but yeah will update